database.go

```diff
@@ -35,6 +36,7 @@ type Recurser struct {
 	isSkippingTomorrow bool
 	schedule           map[string]interface{}
 	isSubscribed       bool
+	currentlyAtRC      bool
```

Comment:
Adding a `currentlyAtRC` field to the Firestore database. This is used in the `endofbatch` cron job to determine if a user should be unsubscribed from pairing bot.